lib/measures/tbd/resources/tbd.rb

```diff
@@ -0,0 +1,34 @@
+begin
+  # Try to load from the Topolys gem.
+  require "topolys"
+
+  puts "... relying on Topolys gem"
```

Comment:
Usually requiring a library would not print output, maybe you could use TBD's logging to log which version was loaded?

Reply:
Correct when assembling TBD as a measure  - no print output. When relying on TBD and/or Topolys as gems (```bundle exec rake update_library_files```), one sees the message fly by. Useful for debugging at the time - will remove.